File: Problems/0344 Reverse String/Solution.js
// https://leetcode.com/problems/reverse-string/
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function (s) {
    let arr = [...s];
    let j = s.length;
    for (let i = 0; i < s.length; i++) {
        j--;
        s[i] = arr[j]
    }
};

let s = ["h", "e", "l", "l", "o"]


console.log(reverseString(s))

console.log(s);

File: Problems/0344 Reverse String/Solution.py
#  https://leetcode.com/problems/reverse-string/
class Solution:
    def reverseString(self, s: list[str]) -> None:
        arr = s.copy()
        j = len(arr)
        for i in range(0, len(arr), 1):
            j -= 1
            s[i] = arr[j]


s = ["h", "e", "l", "l", "o"]
obj = Solution()
obj.reverseString(s)

print(s)
